db/migrate/20230223075432_add_column_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :photo, :string
+    add_column :users, :bio, :string
```

Comment:
- [ ] Great job on creating the tables however the `bio` should be a `text`, not `string`, kindly make sure to change the bio type to text so your migrations reflect the `ERD`